plugins/tileacq.py

```diff
@@ -115,7 +115,8 @@ def __init__(self, microscope, main_app):
 
         self.nx = model.IntContinuous(5, (1, 1000), setter=self._set_nx)
         self.ny = model.IntContinuous(5, (1, 1000), setter=self._set_ny)
-        self.overlap = model.FloatContinuous(20, (1, 80), unit="%")
+        self.overlap = model.FloatContinuous(20, (-200, 500), unit="%")
```

Comment:
Negative range for overlap is actually totally independent of rotation. It's useful (arguably necessary) for CLEM acquisitions where you don't want the EM (smaller field of view) to expose the field of the subsequent FM acquisition. This could also be solved by making it possible to choose which stream you would like to overlap on -- as opposed to defaulting to the stream with the smallest field of view (which makes sense, but happens to not always be what you want).